src/pymmcore_widgets/mda/_core_mda.py

```diff
@@ -54,7 +54,20 @@ def create_subwidgets(self) -> None:
 
 
 class MDAWidget(MDASequenceWidget):
-    """Widget for running MDA experiments, connecting to a MMCorePlus instance."""
+    """A [MDASequenceWidget](../MDASequenceWidget#) Connected to a [`pymmcore_plus.CMMCorePlus`][] instance.
+
+    It provides a GUI to construct and run a [`useq.MDASequence`][].
+
+    Parameters
+    ----------
+    parent : QWidget | None
+        Optional parent widget, by default None.
+    mmcore : CMMCorePlus | None
+        Optional [`CMMCorePlus`][pymmcore_plus.CMMCorePlus] micromanager core.
+        By default, None. If not specified, the widget will use the active
+        (or create a new)
+        [`CMMCorePlus.instance`][pymmcore_plus.core._mmcore_plus.CMMCorePlus.instance].
+    """  # noqa: E501
```

Comment:
remove noqa

Reply:
this is because I want all this line with links as the widget description when making the widget table in the docs

```
[MDASequenceWidget](../MDASequenceWidget#) connected to a [`pymmcore_plus.CMMCorePlus`][] instance
```